CHANGELOG.md

```diff
@@ -3,6 +3,15 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+-
```

Comment:
I'd just remove these lines, we don't have to have this section until there's something to put in it.

Reply:
Yeah, that was an oversight on my part (started putting the changes under Added until I thought better of it). Removed in d73cc7de.